content/hardware/05.pro-solutions/solutions-and-kits/edge-control/tutorials/smart-irrigation-system-v2/content.md

```diff
@@ -0,0 +1,568 @@
+---
+title: 'LoRaWAN® Irrigation System Using Arduino® Edge Control'
+description: "This application note describes how to control a four zones irrigation system using the Edge Control and the Arduino Cloud with LoRaWAN® connectivity"
+difficulty: intermediate
+tags:
+  - Irrigation System
+  - Arduino Edge Control
+  - Arduino MKR WAN 1310
+  - Solenoid Valve
+  - Arduino IoT Cloud
+  - Application Note
+  - WisGate Lite
+  - LoRaWAN®
+author: 'Christopher Mendez'
+libraries:
+  - name: Arduino_EdgeControl
+    url: https://github.com/arduino-libraries/Arduino_EdgeControl
+  - name: ArduinoJson
+    url: https://github.com/bblanchon/ArduinoJson
+  - name: Arduino_ConnectionsHandler
+    url: https://github.com/arduino-libraries/Arduino_ConnectionHandler
+  - name: ArduinoIoTCloud
+    url: https://github.com/arduino-libraries/ArduinoIoTCloud
+software:
+  - ide-v1
+  - ide-v2
+  - web-editor
+  - iot-cloud
+  - arduino-agent
+hardware:
+  - hardware/05.pro-solutions/boards/solutions-and-kits/edge-control
+  - hardware/05.pro-solutions/boards/solutions-and-kits/enclosure-kit
+  - hardware/01.mkr/01.boards/mkr-wan-1310
+---
+
+## Introduction
+
+We know that agricultural activities are normally carried out in remote environments, this makes access to electricity and connectivity a challenge. 
+
+Smart farming techniques are being implemented more and more due to the importance of optimizing the use of resources. This includes the demand for more efficient, eco-friendly, and more profitable crops.
+
+![Application Note Overview. Each pot represents one individual irrigation zone capable of watering a crop field](assets/Thumbnail.png)
```

Comment:
Thank you, it is really important to explain to the reader that this solution can control big irrigation zones without any problem.